addons/NURFG/EditorWidget/TestRunnerDock.cs

```diff
@@ -27,20 +27,20 @@ public override void _Ready()
             InitializeNUnitIfNeeded();
 
             // Connect controls
-            _refreshButton = (Button)FindNode("RefreshButton");
-            _refreshButton.Connect("pressed", this, nameof(RefreshButton_Click));
+            _refreshButton = (Button)GetNode("HBoxContainer/RefreshButton");
+            _refreshButton.Connect("pressed", Callable.From(() => RefreshButton_Click()));
 
-            _runButton = (Button)FindNode("RunButton");
-            _runButton.Connect("pressed", this, nameof(RunButton_Click));
+            _runButton = (Button)GetNode("HBoxContainer/RunButton");
+            _runButton.Connect("pressed", Callable.From(() => RunButton_Click()));
 
-            _resultTree = (Tree)FindNode("ResultTree");
-            _resultTree.Connect("item_selected", this, nameof(TestResultTree_ItemSelected));
-            _resultTree.Connect("item_activated", this, nameof(TestResultTree_ItemActivated));
+            _resultTree = (Tree)GetNode("VSplitContainer/ResultTree");
+            _resultTree.Connect("item_selected", Callable.From(() => TestResultTree_ItemSelected()));
+            _resultTree.Connect("item_activated", Callable.From(() => TestResultTree_ItemActivated()));
 
-            _testOutputLabel = (RichTextLabel)FindNode("TestOutputLabel");
+            _testOutputLabel = (RichTextLabel)GetNode("VSplitContainer/TestOutputLabel");
```

Comment:
Thank you for changing my `FindNode()` calls to `GetNode()`.  As you might have guessed, I wrote this while I was still fairly new to Godot, so I didn't yet know about `FindNode()`'s performance risks.

Reply:
This needed to be ported because Godot 4+ doesn't have FindNode() anymore, doubt that having something like FindNode() in the initialization process will take longer than 2ms, so no real performance risks there.